### PR TITLE
Accepts value for pomodoro interval

### PR DESCRIPTION
**Pull request by stevepentler**
- User is able to enter a time amount for their work time and pomodoro

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const windowEvents = {
     mainWindow.setResizable(false);
     mainWindow.setMovable(false);
     mainWindow.setMinimizable(false);
-    mainWindow.setClosable(false);
+    mainWindow.setClosable(true);
```
Comment (stevepentler) [inline lib/main.js:32]: This will save us some time.

File: lib/render-timer.js [hunk 2]
```diff
@@ -6,20 +6,28 @@ const RenderTimer = React.createClass({
...
     let activeDisplay = <h3>{this.props.timerSeconds} seconds remaining</h3>
     let inactiveDisplay =
           <div className="time-interval-form">
-            <h3>Set Time Interval</h3>
+            <h3>Set Work Interval</h3>
             <form>
               <input
                defaultValue= '30'
-               ref='timer'
+               ref='workTimer'
```
Comment (stevepentler) [inline lib/render-timer.js:21]: Define which form

File: lib/renderer.js [hunk 3]
```diff
@@ -19,20 +19,25 @@ class Application extends Component {
 
   componentDidMount() {
     timer.on('tick', timerState => {
-      this.setState({timerSeconds: timerState.seconds, timerActive: timerState.active})
-      this.assessPomo();
+      this.setState({timerSeconds: this.determineTimeSource(timerState),
```
Comment (stevepentler) [inline lib/renderer.js:22]: Need to know if countdown starts with pomo value or work value

Comment (Jlawlzz) [inline lib/renderer.js:22]: Both

**Comment by Jlawlzz** [comment]
Looks good! I was next you coding with you so if anyone knew, ITD BE ME!

**Pull request merged**